packages/compiler/shared/types.ts

```diff
@@ -131,6 +131,9 @@ export declare function convertToTSX(input: string, options?: ConvertToTSXOption
 // Works in browser: yes ("options" is required)
 export declare function initialize(options: InitializeOptions): Promise<void>;
 
+// Clears internal cache to release memory usage
+export declare function teardown(): void;
```

Comment:
Adding a small example to show how users can use the `teardown` function would be great. While the test is helpful, it's unlikely that users will dive deep into a test to see how and when the `teardown` function should be used.

Reply:
Yep, good call @ematipico! Our compiler docs are quite lacking, but it would be good to start improving these incrementally.